driver/main_args.ml

```diff
@@ -148,6 +148,11 @@ let mk_i f =
 let mk_I f =
   "-I", Arg.String f, "<dir>  Add <dir> to the list of include directories"
 
+let mk_H f =
+  "-H", Arg.String f,
+  "<dir>  Add <dir> to the list of \"hidden\" include directories\n\
+ \     (Like -I, but the program can not directly reference these dependencies)"
```

Comment:
I think is is differently indented by one space compared to the similar entries in this file. Presumably that might make the message indentation off by one too.

Reply:
The indentation of multiple-line descriptions is a bit inconsistent in this file, but I count six flags where this indentation (5 spaces) is used for subsequent: `-opaque`, `-strict-formats`, `-no-strict-formats`, `-for-pack`, `-w`, `-warn-error`.  This makes it the most common choice.  I checked just now, and it prints like them.

Reply:
Note: the absence of printing difference comes from the combination of escaped newline and escaped space:

The following two string literals have the same value:
```
let s1 = "foo\n\
         \  bar"

  let s2 = "foo\n\
           \  bar"
```